release/nightly_tests/dataset/map_benchmark.py

```diff
@@ -48,17 +59,28 @@ def benchmark_fn():
             ds = ds.map(increment_row)
         elif args.api == "map_batches":
             if not args.compute or args.compute == "tasks":
-                ds = ds.map_batches(increment_batch, batch_format=args.batch_format)
+                ds = ds.map_batches(
+                    functools.partial(
+                        increment_batch,
+                        map_batches_sleep_ms=args.map_batches_sleep_ms,
+                    ),
+                    batch_format=args.batch_format,
+                )
             else:
                 ds = ds.map_batches(
                     IncrementBatch,
+                    fn_constructor_args=[args.map_batches_sleep_ms],
                     batch_format=args.batch_format,
                     concurrency=(1, 1024),
                 )
         elif args.api == "flat_map":
             ds = ds.flat_map(flat_increment_row)
 
-        # Iterate over the results.
+        def dummy_write(batch):
+            return {"num_rows": [len(batch["column00"])]}
+
+        ds = ds.map_batches(dummy_write)
```

Comment:
What's the motivation for adding this?

Reply:
* To repro the issue that cluster isn't scaling up due to low utilization, which is caused by reserved resources of idle ops. 
* Make the pipeline DAG similar to the real use cases.